common/rdm/PidStoreTest.cpp

```diff
@@ -409,7 +409,7 @@ void PidStoreTest::testPidStoreDirectoryLoad() {
 
   vector<const PidDescriptor*> all_pids;
   esta_store->AllPids(&all_pids);
-  OLA_ASSERT_EQ(static_cast<size_t>(4), all_pids.size());
+  OLA_ASSERT_EQ(static_cast<size_t>(6), all_pids.size());
```

Comment:
This change and the test data change due to PidStore.py requiring DEVICE_INFO, QUEUED_MESSAGES, and SUPPORTED_PARAMETERS to be defined when loading the pidstore

Reply:
Do you actually mean PidStoreTest.py? i.e. you've written your tests to test against real PIDs and then added them to the test data?

Reply:
Also PROXIED_DEVICE_COUNT, PROXIED_DEVICES and COMMS_STATUS from the looks of things?

Reply:
yes, I copied some of the real pid data into the pid test data so the python sanity checks were satisfied.  I made that change to the cpp tests so they accepted the new test data, but didn't add the same sanity checks to the cpp code.

Reply:
Ah great, thanks for confirming.